### PR TITLE
feat(handle): Add `blurUnselectDisabled` property to disable unselecting handle on blur.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #8485

## Summary

- Adds `blurUnselectDisabled` property
- This is needed so that the element is not deactivated after blur when moving an item via keyboard.
- Adds test for functionality

### CODE REVIEW

**Comment by driskull** [comment]
@benelan can you review this one?

File: packages/calcite-components/src/components/handle/handle.tsx [hunk 1]
```diff
@@ -106,6 +106,13 @@ export class Handle implements LoadableComponent, T9nComponent, InteractiveCompo
    */
   @Prop() label: string;
 
+  /**
+   * When `true`, disables unselecting the component when blurred.
+   *
+   * @internal
+   */
+  @Prop() blurUnselectDisabled = false;
```
Comment (macandcheese) [inline packages/calcite-components/src/components/handle/handle.tsx:114]: Is there a way to tie this name to `activated`, or would that not be accurate? Just a nitpick as its an internal prop, but its a bit confusing if unfamiliar with the component.

File: packages/calcite-components/src/components/handle/handle.e2e.ts [hunk 2]
```diff
@@ -37,6 +37,51 @@ describe("calcite-handle", () => {
     expect(await handle.getProperty("activated")).toBe(true);
   });
 
+  it("sets activated to false when blurred", async () => {
```
Comment (macandcheese) [inline packages/calcite-components/src/components/handle/handle.e2e.ts:40]: Does this fix any behavior in list or another consuming component? If so, is it worth adding a check there? Disregard if not the case.

**Review by macandcheese — APPROVED** [review]
🏆 Nice! Just a couple nitpick / q's.

File: packages/calcite-components/src/components/handle/handle.tsx (hunk 1 above)
Comment (driskull) [reply packages/calcite-components/src/components/handle/handle.tsx:114]: I was thinking that but I think the next step is going to be renaming `active` to `selected` and making it public.

**Pull request merged**